### PR TITLE
[Bugfix](load) fix be will coredump when parsing malformed json file using simdjson

### LINKED ISSUE #12061: [Bug] be will coredump when parsing malformed json file using simdjson
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

master

### What's Wrong?
```
============ terminate called after throwing an instance of 'simdjson::simdjson_error'
  what():  The JSON element does not have the requested type.
*** Query id: 0-0 ***
*** Aborted at 1661344196 (unix time) try "date -d @1661344196" if you are using GNU date ***
*** Current BE git commitID: f87568434 ***
*** SIGABRT unkown detail explain (@0x1fa0000ec4f) received by PID 60495 (TID 0x7fa5c5b79700) from PID 60495; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/common/signal_handler.h:420
 1# 0x00007FA78AEFB570 in /lib64/libc.so.6
 2# gsignal in /lib64/libc.so.6
 3# __GI_abort in /lib64/libc.so.6
 4# __gnu_cxx::__verbose_terminate_handler() [clone .cold] in /home/disk6/yang/doris/be/lib/palo_be
 5# __cxxabiv1::__terminate(void (*)()) in /home/disk6/yang/doris/be/lib/palo_be
 6# 0x00007FA7A325E4F1 in /home/disk6/yang/doris/be/lib/palo_be
 7# 0x00007FA7A325E645 in /home/disk6/yang/doris/be/lib/palo_be
 8# std::ostream& simdjson::operator<< <simdjson::haswell::ondemand::raw_json_string>(std::ostream&, simdjson::simdjson_result<simdjson::haswell::ondemand::raw_json_string>) at /home/disk6/yang/workspace/baidu/bdg/doris/core/thirdparty/installed/include/simdjson/error.h:292
 9# doris::vectorized::VSIMDJsonReader::_vhandle_simple_json(doris::vectorized::Block&, std::vector<doris::SlotDescriptor*, std::allocator<doris::SlotDescriptor*> > const&, bool*, bool*) at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/vec/exec/vjson_scanner.cpp:579
10# doris::vectorized::VSIMDJsonReader::read_json_column(doris::vectorized::Block&, std::vector<doris::SlotDescriptor*, std::allocator<doris::SlotDescriptor*> > const&, bool*, bool*) at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/vec/exec/vjson_scanner.cpp:559
11# doris::vectorized::VJsonScanner<doris::vectorized::VSIMDJsonReader>::get_next(doris::vectorized::Block*, bool*) at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/vec/exec/vjson_scanner.cpp:68
12# doris::vectorized::VBrokerScanNode::scanner_scan(doris::TBrokerScanRange const&, doris::ScannerCounter*) at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/vec/exec/vbroker_scan_node.cpp:232
13# doris::vectorized::VBrokerScanNode::scanner_worker(int, int) at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/vec/exec/vbroker_scan_node.cpp:283
14# doris::vectorized::VBrokerScanNode::start_scanners()::{lambda()#1}::operator()() const at /home/disk6/yang/workspace/baidu/bdg/doris/core/be/src/vec/exec/vbroker_scan_node.cpp:105
15# void std::__invoke_impl<void, doris::vectorized::VBrokerScanNode::start_scanners()::{lambda()#1}>(std::__invoke_other, doris::vectorized::VBrokerScanNode::start_scanners()::{lambda()#1}&&) at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/invoke.h:61
16# std::__invoke_result<doris::vectorized::VBrokerScanNode::start_scanners()::{lambda()#1}>::type std::__invoke<doris::vectorized::VBrokerScanNode::start_scanners()::{lambda()#1}>(doris::vectorized::VBrokerScanNode::start_scanners()::{lambda()#1}&&) at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/invoke.h:97
17# void std::thread::_Invoker<std::tuple<doris::vectorized::VBrokerScanNode::start_scanners()::{lambda()#1}> >::_M_invoke<0ul>(std::_Index_tuple<0ul>) at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/std_thread.h:253
18# std::thread::_Invoker<std::tuple<doris::vectorized::VBrokerScanNode::start_scanners()::{lambda()#1}> >::operator()() at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/std_thread.h:260
19# std::thread::_State_impl<std::thread::_Invoker<std::tuple<doris::vectorized::VBrokerScanNode::start_scanners()::{lambda()#1}> > >::_M_run() at /home/disk6/yang/workspace/baidu/bdg/doris/palo-toolchain/ldb_toolchain/include/c++/11/bits/std_thread.h:211
20# execute_native_thread_routine in /home/disk6/yang/doris/be/lib/palo_be
21# start_thread in /lib64/libpthread.so.0
22# __clone in /lib64/libc.so.6
```
### What You Expected?

not core

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by yangzhg**
# Proposed changes

Issue Number: close #12061 

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [x] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: regression-test/suites/load_p0/stream_load/test_json_load.groovy [hunk 1]
```diff
@@ -63,6 +63,33 @@ suite("test_json_load", "p0") {
         assertTrue(result2[0].size() == 1)
         assertTrue(result2[0][0] == 1, "Insert should update 1 rows")
     }
+
+    def test_invalid_json_array_table = { testTablex ->
+        // multi-line sql
+        def result1 = sql """
+                        CREATE TABLE IF NOT EXISTS ${testTable} (
```
Comment (carlvinhust2012) [inline regression-test/suites/load_p0/stream_load/test_json_load.groovy:70]: this line is repeated.

Comment (yangzhg) [reply regression-test/suites/load_p0/stream_load/test_json_load.groovy:70]: done

**Review by eldenmoon — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**